sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -100,7 +100,10 @@ namespace Azure { namespace Core {
        *
        * @remark The Proxy Information string is composed of a set of elements
        * formatted as follows:
+       *
+       * @code
        * (\[\<scheme\>=\]\[\<scheme\>"://"\]\<server\>\[":"\<port\>\])
+       * @endcode
```

Comment:
Previously, on doxygen it looked like this:
https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-core/1.10.3/struct_azure_1_1_core_1_1_http_1_1_win_http_transport_options.html#aaee8369faa29d288f349fc88e04a1462

![image](https://github.com/Azure/azure-sdk-for-cpp/assets/6527137/2470770a-14d7-4eb3-8e99-a2c517562c37)

Presumably, that view is also helped a bit by this change (certainly won't hurt it)

Reply:
Agreed. That's why I chose to fix it in the source rather than attempt to work around it in the ApiView tool.